docs/03-pages/01-building-your-application/06-configuring/10-custom-server.mdx

```diff
@@ -88,10 +88,17 @@ const app = next({})
 
 The above `next` import is a function that receives an object with the following options:
 
-- `dev`: `Boolean` - Whether or not to launch Next.js in dev mode. Defaults to `false`
-- `dir`: `String` - Location of the Next.js project. Defaults to `'.'`
-- `quiet`: `Boolean` - Hide error messages containing server information. Defaults to `false`
-- `conf`: `object` - The same object you would use in [next.config.js](/docs/pages/api-reference/next-config-js). Defaults to `{}`
+|Option|Type|Description|
```

Comment:
Thank you! Could we run Prettier here?

Reply:
@leerob 
I thought I had, so thanks for keeping me honest. I believe this is ready to go now.